terraform/environments/mlra/main.tf

```diff
@@ -0,0 +1,49 @@
+module "lb-access-logs-enabled" {
+  source = "github.com/ministryofjustice/modernisation-platform-terraform-loadbalancer"
+  providers = {
+    aws.bucket-replication = aws
+  }
+
+  vpc_all = local.vpc_all
+  #existing_bucket_name               = "my-bucket-name"
```

Comment:
Is this required? Not having this input implies that the module should create an S3 bucket to drop ALB Access Logs to. Inserting implies we have an S3 bucket already that we would like to use, and so the module shouldn't create a bucket for us